.github/workflows/worker.yml

```diff
@@ -0,0 +1,36 @@
+name: worker
+on:
+  push:
+    branches:
+      - "master"
+      - "worker/**"
+
+jobs:
+  binary:
+    runs-on: windows-latest
+
+    steps:
+      - name: 🐙 Checkout GitHub repo (+ download lfs dependencies)
+        uses: actions/checkout@v3
+        with:
+          fetch-depth: 0
+
+      - name: 🐍 Set up Python
+        uses: actions/setup-python@v2
```

Comment:
The last version is `actions/setup-python@v5`. See: https://github.com/actions/setup-python